src/app/components/common/alert-dialog/alert-dialog.component.ts

```diff
@@ -107,8 +113,8 @@ export class AlertDialogComponent implements OnInit, OnChanges {
         this.delete.emit(this.childData.id);
         this.toggleModal(false);
       }
-      if (this.childData.key === 'deactivateUser') {
-        this.deleteUserService.emit(this.childData.id);
+      if (['deactivateUser', 'enableUser', 'unblockUser', 'clearSessions'].includes(this.childData.key)) {
```

Comment:
enable or activate

We are using deactivate term to disable?!

Reply:
> We are using deactivate term to disable?!
Yes